crypto/bn/bn_lib.c

```diff
@@ -430,42 +430,92 @@ int BN_set_word(BIGNUM *a, BN_ULONG w)
     return 1;
 }
 
-BIGNUM *BN_bin2bn(const unsigned char *s, int len, BIGNUM *ret)
+typedef enum {big, little} endianess_t;
+typedef enum {sign, unsign} signedness_t;
```

Comment:
Could better names be used for the enum values?

At the least have them in uppercase so they look like a constant.

Better might be: BIG, LITTLE, SIGNED & UNSIGNED.
Better again: E_BIG, E_LITTLE, E_SIGNED, E_UNSIGNED.

Reply:
The `E_` prefix makes me think of errors, so I'd prefer not...  but I have nothing against a little capitalization :wink:

Reply:
Capitals works for me.

Reply:
Done